components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/utils/mappings/APIMappingUtil.java

```diff
@@ -1394,13 +1412,21 @@ private static APIOperationsDTO getOperationFromURITemplate(URITemplate uriTempl
      *
      * @return a default operations list
      */
-    private static List<APIOperationsDTO> getDefaultOperationsList() {
+    private static List<APIOperationsDTO> getDefaultOperationsList(String apiType) {
 
         List<APIOperationsDTO> operationsDTOs = new ArrayList<>();
+        String[] suportMethods = null;
+
+        if(apiType.equals(APIConstants.GRAPHQL_API)) {
```

Comment:
Formatting issue and swap equals condition.

Reply:
fixed